conjur/cli.py

```diff
@@ -231,6 +292,34 @@ def run_client_action(resource, args):
             # A successful exit is required to prevent the initialization of
             # the Client because the init command does not require the Client
             return
+        elif resource == 'login':
+            credential_data = CredentialData(None, args.name, None)
+            credentials = CredentialsFromFile()
+            login_logic = LoginLogic(credentials)
+            login_controller = LoginController(ssl_verify=args.ssl_verify,
+                                               password=args.password,
+                                               credential_data=credential_data,
+                                               login_logic=login_logic)
+            login_controller.load()
+            return
+        elif resource == 'logout':
+            credentials = CredentialsFromFile()
+            logout_logic = LogoutLogic(credentials)
+
+            logout_controller = LogoutController(ssl_verify=None,
+                                                 logout_logic=logout_logic)
+            logout_controller.remove_credentials()
+            return
+        # pylint: disable=line-too-long
+        if not os.path.exists(DEFAULT_NETRC_FILE) or (os.path.exists(DEFAULT_NETRC_FILE) and os.path.getsize(DEFAULT_NETRC_FILE) == 0):
+            credential_data = CredentialData(None, None, None)
```

Comment:
Put `None` as default values on the parameters, so you won't have to explicitly pass them here.

Reply:
**much** cleaner. Thanks for the suggestion